save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -50,12 +52,23 @@ class Contest(
     /**
      * @return set of testSuiteIds
      */
-    fun getTestSuiteIds() = testSuiteIds.split(",")
+    fun getTestSuiteIds() = testSuiteIds.split(DELIMITER)
         .mapNotNull {
             it.toLongOrNull()
         }
         .toSet()
 
+    private fun isTestSuiteIdsValid() = testSuiteIds.isBlank() || testSuiteIds.all { it.isDigit() || it.toString() == DELIMITER }
```

Comment:
`isBlank` allows white spaces, do we consider this valid?

Reply:
Blank `testSuiteIds` field might mean that there are no test suites connected to the contest yet. But actually it is the same as empty here

Reply:
Then why not use `isEmpty` so that we won't accidentally allow blank string in the table?